gluten-flink/docs/Flink.md

```diff
@@ -82,7 +82,7 @@ mvn test
 ## Dependency library deployment
 
 You need to get the Velox4j packages and used them with gluten.
-Velox4j jar available now is velox4j-0.1.0-SNAPSHOT.jar. 
+Velox4j jar available now is under velox4j project target/velox4j-0.1.0-SNAPSHOT.jar. 
```

Comment:
This "Dependency library deployment" section looks unnecessary. The jar deployment is clear according to "Flink local cluster" section. I think we can remove the above section.